config/locales/et.yml

```diff
@@ -11,6 +11,7 @@ et:
   created: "Loodud."
   updated: "Uuendatud."
   deleted: "Kustutatud."
+  something_went_wrong: "Midagi läks valesti."
```

Comment:
The intention is to translate exiting key from `en` locale? Or how is it connected to this PR?